sys/kern/systrace_args.c

```diff
@@ -3457,7 +3457,8 @@ systrace_args(int sysnum, void *params, uint64_t *uarg, int *n_args)
 		struct dbquery_args *p = params;
 		uarg[a++] = (intptr_t)p->query; /* const char * */
 		iarg[a++] = p->querylen; /* int */
-		uarg[a++] = (intptr_t)p->buf; /* char * */
+		uarg[a++] = (intptr_t)p->data; /* char * */
+		uarg[a++] = (intptr_t)p->headers; /* char * */
 		iarg[a++] = p->reslen; /* int */
 		*n_args = 4;
```

Comment:
This would need to be 5, since we added a new argument.

Reply:
Thank you, Stelios! Fixed.